### DISCUSSION TITLE
test service installation error

### DISCUSSION THREAD

**Discussion by AmirMoshfeghi** [linkerd/linkerd2#7504] [Q&A] [answered]
Hi, I am trying Linkerd multicluster for the first time while going through the test service Installation guide.
https://linkerd.io/2.11/tasks/multicluster/#installing-the-test-services 

Unfortunately while doing the "Intalling the test services" section, I get an error as such,

```
error: accumulating resources: accumulation err='accumulating resources from '../base': '/tmp/kustomize-996954789/multicluster/base' must resolve to a file': recursed accumulation of path '/tmp/kustomize-996954789/multicluster/base':no matches for Id apps_v1_Deployment|test|podinfo; failed to find unique target for patch apps_v1_Deployment|podinfo
Error from server (NotFound): deployments.apps "podinfo" not found
```

Could you please help with this issue? thanks

**Comment by alpeb** [comment, marked as answer]
Thanks for the report; there are indeed some issues with the tutorial. Will post a fix soon.

**Reply by alpeb** [reply]
The tutorial should be fixed by now. Please give it another try and let us know how it goes!

**Reply by AmirMoshfeghi** [reply]
Yes, it's fixed. Thanks a lot for the support.
